### PR TITLE
Crazy egg 91446470

### PR DESCRIPTION
**Pull request by ceckert**
Add Crazy Egg for SEO.

### CODE REVIEW

**Comment by ceckert** [comment]
@thescubageek  @levibrown, can you review please?

File: public/static/components/meta/edit.html [hunk 1]
```diff
@@ -50,6 +50,25 @@
              placeholder="http://favicon_url/icon.ico"
               />
   </div>
+  <div class="form-field check-field">
+      {{ widget.use_crazy_egg.id_hidden_field }}
+      <input type="hidden"
```
Comment (thescubageek) [inline public/static/components/meta/edit.html]: How is this hidden field being used?

**Comment by dreuss** [comment]
Look good, let'r rip!

File: public/static/components/meta/edit.html [hunk 2]
```diff
@@ -50,6 +50,25 @@
              placeholder="http://favicon_url/icon.ico"
               />
   </div>
+  <div class="form-field check-field">
```
Comment (thescubageek) [inline public/static/components/meta/edit.html]: @ceckert I'm going to assume this voodoo works...

**Pull request merged**